README.md

```diff
@@ -96,5 +96,5 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/atomic
 The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
 
 
-### Things Not Quite Implemented
-People, Groups, Sets, Blogs
+## Things Not Quite Implemented
+People, Sets
```

Comment:
That's probably what they are referring to actually. I think you could be right. I think it would be fine to remove it because we can't remember what exactly it's referring to. And then we can add things as we find them again.